src/src/radio.c

```diff
@@ -364,7 +356,12 @@ static otError radioProcessTransmitSecurity(otRadioFrame *aFrame)
         otMacFrameSetFrameCounter(aFrame, sMacFrameCounter++);
     }
 
+#if defined(_SILICON_LABS_32B_SERIES_2) && OPENTHREAD_RADIO
+    efr32PlatProcessTransmitAesCcm(aFrame, &sExtAddress);
```

Comment:
Where is efr32PlatProcessTransmitAesCcm defined in GitHub?

Reply:
I have added `ieee802154-packet-utils.cpp` to `to-efr32` to enable this support.

Reply:
@romacdon I have removed reference to `efr32PlatProcessTransmitAesCcm` and related cpp file.